docs/src/er_model.plantuml

```diff
@@ -22,21 +40,33 @@ entity "User Expense" as uexp {
   * id : number <<generated>>
   ---
   * value : number
-  * name: string
-  description: string
+  * name : string
+  * description : string
+  * startDate : date
   ---
```

Comment:
Shouldn't this also have a nullable predefined_expense_id?

Reply:
Good catch. Yes, I forgot this one.